sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1134,28 +1119,44 @@ _pi_queue::_pi_queue(std::vector<ze_command_queue_handle_t> &ComputeQueues,
   CopyCommandBatch.QueueBatchSize = ZeCommandListBatchCopyConfig.startSize();
 }
 
+static pi_result cleanup(pi_event);
```

Comment:
Why did you change this to be a free function instead of being a member of `_pi_event`?